run_testlist.py

```diff
@@ -58,6 +58,155 @@
                   }
                  ]
 
+ec_full_strip_testlist = [{'testcase': 'ec_ior_full_stripe_EC_2P1G1',
+                           # Number of servers, number of clients,
+                           # timeout in minutes
+                           'testvariants': [(4, 8, 5)],
+                           'ppc': 32,
+                           'env_vars': {
+                               'chunk_size': '33554432',
```

Comment:
I'm not yet familiar with EC. How do we choose this number for the chunk_size?

Reply:
Right now we calculate manually but will have some formula in place and will add documents too so others can calculate as well.

Reply:
This isn't fully detailed, but here is the test plan for Frontera:
https://wiki.hpdd.intel.com/pages/viewpage.action?spaceKey=DAOS&title=Frontera+Test+Plan

Reply:
Updated wiki test plan for both the cases